change/@ni-nimble-blazor-92675eea-194d-4c8d-81ea-ff1049cd4f6c.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
Since we are modifying the scripts they depend on make sure to manually test the Blazor server and Blazor Hybrid example apps locally for regressions. They are not tested on CI.